source/docs/hardware/hardware-basics/status-lights-ref.rst

```diff
@@ -6,202 +6,511 @@ Many of the components of the FRC Control System have indicator lights that can
+| Blinking Green         | Successful Calibration |
++------------------------+------------------------+
+| Blinking Red           | Failed Calibration     |
++------------------------+------------------------+
+
++--------------------------------------------------------------------------------------------+
+| Blink Codes During Normal Operation                                                        |
++==============================+================+============================================+
+| LEDs                         | Colors         | Talon SRX State                            |
++------------------------------+----------------+--------------------------------------------+
+| Both                         | Blinking Green | Forward throttle is applied.               |
+|                              |                | Blink rate is proportional to Duty Cycle.  |
++------------------------------+----------------+--------------------------------------------+
+| Both                         | Blinking Red   | Reverse throttle is applied.               |
+|                              |                | Blink rate is proportional to Duty Cycle.  |
++------------------------------+----------------+--------------------------------------------+
+| None                         | None           | No power is being applied to Talon SRX     |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Orange     | CAN bus detected, robot disabled           |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Slow Red   | CAN bus/PWM is not detected                |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Fast Red   | Fault Detected                             |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Red/Orange     | Damanged Hardware                          |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Strobe towards (M-)     | Off/Red        | Forward Limit Switch or Forward Soft Limit |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Strobe towards (M+)     | Off/Red        | Reverse Limit Switch or Reverse Soft Limit |
++------------------------------+----------------+--------------------------------------------+
+| LED1 Only (closest to M+/V+) | Green/Orange   | In Boot-loader                             |
++------------------------------+----------------+--------------------------------------------+
+
++----------------------------------------+
+| B/C CAL Blink Codes                    |
++======================+=================+
+| B/C CAL Button Color | Talon SRX State |
```

Comment:
Make this row the header row and move the row above to be a section title

Reply:
Fixed in c172d27.